bundles/org.openhab.ui/web/src/components/theme-switcher.vue

```diff
@@ -74,14 +74,23 @@
             <span v-t="'about.miscellaneous.webaudio.enable'" />
             <f7-toggle :checked="webAudio === 'enabled'" @toggle:change="setWebAudio" />
           </f7-list-item>
+          <f7-list-item>
+            <span v-t="'about.miscellaneous.commandItem.title'" />
+            <item-picker title="Item" :multiple="false" :value="commandItem" @input="setCommandItem" />
+          </f7-list-item>
```

Comment:
Also it's not looking too good on narrow screens:

![image](https://github.com/openhab/openhab-webui/assets/2004147/53209c93-fda4-47fd-b72a-3bf8c2e9e826)

I think we could just have the item-picker component directly added to the list with an appropriate label like "Listen to UI commands from", instead of the item-picker inside a list item (the item-picker is a list item itself).